src/Prima/Pyxis/Link.elm

```diff
@@ -275,63 +270,80 @@ withClass class_ =
     addOption (Class class_)
 
 
-{-| Represent an icon from Pyxis Iconset.
+{-| Adds classes to the `classList` of the `Link`.
 -}
-type Icon
-    = ArrowRight
-    | ChevronDown
-    | Download
-    | Edit
-    | Email
-    | Phone
+withClassList : List ( String, Bool ) -> Config msg -> Config msg
+withClassList classList =
+    addOption (ClassList classList)
 
 
-{-| Creates a arrow-right icon.
+{-| Adds a generic attribute to the Link.
+-}
+withAttribute : Html.Attribute msg -> Config msg -> Config msg
+withAttribute attr =
+    addOption (Attribute attr)
+
+
+{-| Adds an `onClick` Html.Event to the `Link`.
+-}
+withOnClick : msg -> Config msg -> Config msg
+withOnClick tagger =
+    addOption (Event (H.stopEvt "click" tagger))
+
+
+{-| Adds an `onMouseDown` Html.Event to the `Link`.
 -}
-withIconArrowRight : Config msg -> Config msg
-withIconArrowRight (Config linkConfig) =
-    Config { linkConfig | icon = Just ArrowRight }
+withOnMouseDown : msg -> Config msg -> Config msg
```

Comment:
come mai su una link c'e' la necessita di avere questo handle?
in che caso d'uso lo vedresti bene?

Reply:
In realtà, penso che click potrebbe bastare, era per averli uniformi, ne parlavo con @kioan000. Hover / leave invece me li vedo abbastanza bene.